src/test/java/teammates/test/cases/ui/StudentHomePageActionTest.java

```diff
@@ -160,7 +160,7 @@ public void testExecuteAndPostProcess() throws Exception{
         a = getAction(submissionParams);
         r = getShowPageResult(a);
         data = (StudentHomePageData) r.data;
-        assertEquals(1, data.courses.size());
+        assertEquals(1, data.getCourseTables().size());
         
         
         // Delete additional sessions that were created
```

Comment:
I was expecting to see some new tests. Is the coverage good? I think we should add a `*PageDataTest` class.